x-pack/legacy/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -200,6 +201,7 @@ const getAuthenticationColumns = (): AuthTableColumns => [
         />
       );
     },
+    width: '5%',
```

Comment:
The `5%` width gets a bit too narrow at smaller viewport sizes, causing text truncation for header cells and line breaks for body cells (especially if there are more than 4 digits in the count). I'd recommend changing this and all other changed instances of `5%` in this PR to something like `7%` or `8%` to be on the safe side.

Reply:
Set it to `8%`. How does this look?
<img width="1476" alt="Screen Shot 2019-10-22 at 3 34 30 PM" src="https://user-images.githubusercontent.com/6935300/67335822-8eac8280-f4e1-11e9-8037-d379201bb955.png">
<img width="1472" alt="Screen Shot 2019-10-22 at 3 34 18 PM" src="https://user-images.githubusercontent.com/6935300/67335823-8f451900-f4e1-11e9-8e28-f8b734494217.png">

Reply:
Works for me. Thanks!